### PR TITLE
Updated  /kubernetes page with new copy

### LINKED ISSUE #4292: copy updates /kubernetes
copydoc https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.xnpkx4ypduk

### LINKED ISSUE #4293: copy updates /support/plans-and-pricing
copydoc https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.l83dv5879ajz

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated k8s index page with new copy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/kubernetes](http://0.0.0.0:8001/kubernetes)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc k8s](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#) and [copy doc plans-and-pricing](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.j34bveun4fte) and accept all suggestions

## Issue / Card

Fixes #4292 and fixes #4293

### CODE REVIEW

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -28,7 +19,64 @@ <h1>Multi-cloud Kubernetes</h1>
...
+        <li class="p-list__item is-ticked">Often used for hand-built clusters with custom operations regimes.</li>
+      </ul>
+      <p><a href="/kubernetes/contact-us?product=kubernetes-kubeadm">Get support for kubeadm&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <h2>Getting started with Kubernetes consulting</h2>
```
Comment (Caleb-Ellis) [inline templates/kubernetes/index.html]: `consulting` => `consulting packages`

File: templates/support/plans-and-pricing.html [hunk 2]
```diff
@@ -187,15 +187,16 @@ <h2>Kubernetes</h2>
 <section class="p-strip--light is-deep is-bordered">
   <div class="row">
     <div class="col-12">
-      <h2 class="u-no-margin--bottom">Kubernetes services packages</h2>
+      <h2 class="u-no-margin--bottom">Kubernetes consulting and training packages</h2>
```
Comment (Caleb-Ellis) [inline templates/support/plans-and-pricing.html]: Looks weird with no margin-bottom?

File: templates/shared/pricing/_kubernetes-consulting.html [hunk 3]
```diff
@@ -24,9 +20,9 @@ <h2 class="p-heading--four">Kubernetes Explorer</h2>
   <div class="col-6 p-card">
     <div class="p-card__header">
       <h2 class="p-heading--four">Kubernetes Discoverer</h2>
-      <p><span class="p-heading--two">$45,000</span> one off fee</p>
+      <p><span class="p-heading--two">$45,000</span><br />Workshop and custom architecture</p>
     </div>
-    <p>Get up and running in one week with a customised architecture to fit your requirements, including deploying on virtualised environments, private and public clouds.</p>
+    <p>Week-long training and workshop together with design of a customised architecture to fit your requirements, including deploying on virtualised environments, private and public clouds.</p>
```
Comment (Caleb-Ellis) [inline templates/shared/pricing/_kubernetes-consulting.html]: There's a bit missing at the bottom: `Optional add on: AI/ML $40,000 one off fee`. Not sure if it should be removed in the copydoc or included here

File: templates/kubernetes/index.html [hunk 4]
```diff
@@ -37,7 +85,7 @@ <h2>Kubernetes services packages</h2>
   <div class="row">
     <div class="col-12">
       <h3>AI/ML Add-on for Discoverer and Discoverer Plus</h3>
-      <p>Turn on the taps with a workshop to understand the full stack of machine learning. Build a full pipeline from developer stations to your data centre, to the public cloud. Canonical works with the leading companies to ensure you have the widest range of choices.</p>
+      <p>Extend your K8s workshop to understand the full stack of machine learning tooling and operations. Build a full AI analytics pipeline from developer workstations to your data centre and, to the public cloud. GPGPU integration, public cloud optimisation and tools like Kubeflow and Tensorflow are covered. Workshop includes an assessment by an independent data scientist. Canonical works with the leading hardware, cloud and data science companies to ensure you have the widest range of choices for integration in your enterprise AI strategy.</p>
```
Comment (Caleb-Ellis) [inline templates/kubernetes/index.html]: `and, to the public cloud` => `and the public cloud`

File: templates/kubernetes/index.html [hunk 5]
```diff
@@ -56,47 +104,47 @@ <h3>AI/ML Add-on for Discoverer and Discoverer Plus</h3>
 
 <section class="p-strip is-bordered">
   <div class="row">
-    <h2>Why choose the Canonical distribution?</h2>
-    <p>Google, Microsoft, Amazon and many other institutions run Kubernetes on Ubuntu because we focus on the latest container capabilities in modern kernels. That’s why it’s the top choice for enterprise Kubernetes, too.</p>
+    <h2>Ubuntu is the leading platform for Kubernetes</h2>
+    <p>Google, Microsoft, Amazon, Cisco, IBM and others run cloud Kubernetes services on Ubuntu because we focus on the latest container capabilities in modern kernels. That’s why Ubuntu iit’s the top choice for on-premise enterprise Kubernetes, too, with microk8s, kubeadm and charmed Ubuntu Kubernetes all supported by Canonical.</p>
```
Comment (Caleb-Ellis) [inline templates/kubernetes/index.html]: `iit's` => `is`

**Comment by anasereijo** [comment]
@pmahnke just noticed there are more copy changes added, so I'll review it once this is updated.

**Pull request merged**